### PR TITLE
Update UG format for delete insurance and list claims features

### PR DESCRIPTION
**Pull request by zi-yii**
UG format for delete insurance and list claims features are updated to make the UG clearer.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -325,18 +350,14 @@ Closes a claim from an insurance plan of a client.
 
 Format: `closeClaim INDEX iid/INSURANCE_ID cid/CLAIM_ID`
 
-IDs for insurance plans:
```
Comment (yanqiyqh) [inline docs/UserGuide.md:328]: Is it better to keep this ID reference for insurance plans in each command? Or will it be too repetitive

**Review by yanqiyqh — COMMENTED** [review]
LGTM, just one small comment

File: docs/UserGuide.md (hunk 1 above)
Comment (RezwanAhmed123) [reply docs/UserGuide.md:328]: i'll merge first, if it seems too repetitive, will remove in a future PR

**Review by RezwanAhmed123 — APPROVED** [review]
LGTM for now

**Pull request merged**